README.md

```diff
@@ -73,28 +83,28 @@ the RFC has received ample discussion and enough of the tradeoffs have been
 discussed. The Shepherd Team will propose to either accept or reject the RFC
 after the FCP.
 
+##### RFC Categories
+In order to do do justice to the different aspects of documents that merit
+generation of broad community consensus via the RFC process, we classify each
+RFC into _feature_, _process_ and _informational_ RFCs. All follow the same
+high-level process as described above, but each category requires a different
+"mode of discussion", templates, criteria and judgment that it is beneficial
+to the overall RFC process to identify those categories explicitly.
+
 
 ## Process from Creation to Merge
 
-*In short, to get a major change included in Nix or Nixpkgs, one must
+*In short, to get a major change included in Nix, Nixpkgs or the ecosystem, one must
 first get the RFC merged into the RFC repository as a markdown file under the
-`rfcs` directory. At that point the RFC is accepted and may be implemented
-with the goal of eventual inclusion into Nix or Nixpkgs.*
-
-0. Have a cool idea!
-1. Fill in the RFC. Put care into the details: RFCs that do not present
-   convincing motivation, demonstrate understanding of the impact of the design,
-   or are disingenuous about the drawbacks or alternatives tend to be
-   poorly-received. You might want to create a PR in your fork of the RFCs
-   repository to help you flesh it out with a few supporters or chat/video
-   conference with a few people involved in the topic of the RFC.
-2. In case your RFC is a technical proposal, you might want to prepare a
-   prototype of your idea to firstly make yourself aware of potential pitfalls
-   and also help reviewers understand the RFC. Code may be able to explain some
-   issues in short.
+corresponding directory. At that point the RFC is accepted and may be implemented
+with the goal of eventual inclusion into Nix, Nixpkgs or the Ecosystem.
+
+0. Have a cool idea or an important information!
+1. Identify its category: _feature_, _process_ or _informational_.
+2. Start with the correct template and follow the instructions and comments.
 3. Submit a pull request. As a pull request the RFC will receive design feedback
    from the larger community, and the author should be prepared to revise it in
-   response.
+   response. Leave them in draft while you're still actively working on them.
```

Comment:
This one is a bit contentious too IMO. If RFC stands for Request For Comments, then it should be implied that you're requesting feedback, and that it's not necessarily fully done the moment you post it. So why create draft RFCs at all?

Reply:
There is currently a draft label to denote the draft stage. This is indeed orthogonal to this RFC, but I thought it might help the RFC steering commitee once things scale up (if they do).

I'll remove this as a strictly unrelated change.